modules/core/src/test/scala/com/azavea/stac4s/CatalogLayerSpec.scala

```diff
@@ -218,101 +215,114 @@ class CatalogLayerSpec extends AnyFunSpec with Matchers {
-            _type = `image/tiff`.some
+            _type = `image/tiff`.some,
+            extensionFields = ().asJsonObject
             // "eo:bands": [4],
           ),
           "B6" -> StacItemAsset(
             href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B6.TIF",
             title = "SWIR (B6)".some,
             description = "SWIR Band Top Of the Atmosphere".some,
             roles = Nil,
-            _type = `image/tiff`.some
+            _type = `image/tiff`.some,
+            extensionFields = ().asJsonObject
             // "eo:bands": [5],
           ),
           "B7" -> StacItemAsset(
             href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B7.TIF",
             title = "SWIR Band (B7)".some,
             description = "SWIR Band Top Of the Atmosphere".some,
             roles = Nil,
-            _type = `image/tiff`.some
+            _type = `image/tiff`.some,
+            extensionFields = ().asJsonObject
             // "eo:bands": [6],
           ),
           "B8" -> StacItemAsset(
             href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B8.TIF",
             title = "Panchromatic Band (B8)".some,
             description = "Panchromatic Band Top Of the Atmosphere".some,
             roles = Nil,
-            _type = `image/tiff`.some
+            _type = `image/tiff`.some,
+            extensionFields = ().asJsonObject
             // "eo:bands": [7],
           ),
           "B9" -> StacItemAsset(
             href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B9.TIF",
             title = "Cirrus Band (B9)".some,
             description = "Cirrus Band Top Of the Atmosphere".some,
             roles = Nil,
-            _type = `image/tiff`.some
+            _type = `image/tiff`.some,
+            extensionFields = ().asJsonObject
             // "eo:bands": [8],
           ),
           "B10" -> StacItemAsset(
             href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B10.TIF",
             title = "LWIR Band (B10)".some,
             description = "LWIR Band Top Of the Atmosphere".some,
             roles = Nil,
-            _type = `image/tiff`.some
+            _type = `image/tiff`.some,
+            extensionFields = ().asJsonObject
             // "eo:bands": [9],
           ),
           "B11" -> StacItemAsset(
             href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B11.TIF",
             title = "LWIR Band (B11)".some,
             description = "LWIR Band Top Of the Atmosphere".some,
             roles = Nil,
-            _type = `image/tiff`.some
+            _type = `image/tiff`.some,
+            extensionFields = ().asJsonObject
             // "eo:bands": [10],
           )
         ),
```

Comment:
How about adding into this test a neew layer extension usage example test?
1. It can be an API usage example
2. just a sainity check that it is possible to use codecs and API (that is how I revealed a bad codec)

smth like ~
```scala
ItemExtension[LayerItemExtension].getProperties(item) shouldBe LayerItemExtension(List("layer-us", "layer-ca")).valid
```